server/services/cms.js

```diff
@@ -56,78 +59,77 @@ class CmsService {
-        return this.getVideo(establishmentId, location);
+        return this.getVideo(establishmentName, establishmentId, location);
       /// ...other types go here
       default:
         // log unsupported type
         return null;
     }
   }
 
-  async getAudio(establishmentId, location) {
+  async getAudio(establishmentName, establishmentId, location) {
     const data = await this.#cmsApi.get(new AudioPageQuery(location));
-    const { id, seriesId, episodeId } = data;
+    const { id, seriesId, seriesSortValue } = data;
     const suggestedContent = await this.#contentRepository.suggestedContentFor({
       id,
       establishmentId,
     });
-
-    const filterOutCurrentEpisode = episodes =>
-      episodes.filter(item => item.id !== id);
-
-    const nextEpisodes = await this.#contentRepository.nextEpisodesFor({
-      id: seriesId,
-      establishmentId,
-      perPage: 3,
-      episodeId,
-    });
+    const nextEpisodes = await this.getNextEpisode(
+      establishmentName,
+      seriesId,
+      seriesSortValue,
+    );
 
     return {
       ...data,
       suggestedContent,
-      nextEpisodes: nextEpisodes
-        ? filterOutCurrentEpisode(nextEpisodes)
-        : nextEpisodes,
+      nextEpisodes,
     };
   }
 
-  async getVideo(establishmentId, location) {
+  async getVideo(establishmentName, establishmentId, location) {
     const data = await this.#cmsApi.get(new VideoPageQuery(location));
-    const { id, seriesId, episodeId } = data;
+    const { id, seriesId, seriesSortValue } = data;
     const suggestedContent = await this.#contentRepository.suggestedContentFor({
       id,
       establishmentId,
     });
-
-    const filterOutCurrentEpisode = episodes =>
-      episodes.filter(item => item.id !== id);
-
-    const nextEpisodes = await this.#contentRepository.nextEpisodesFor({
-      id: seriesId,
-      establishmentId,
-      perPage: 3,
-      episodeId,
-    });
+    const nextEpisodes = await this.getNextEpisode(
+      establishmentName,
+      seriesId,
+      seriesSortValue,
+    );
 
     return {
       ...data,
       suggestedContent,
-      nextEpisodes: nextEpisodes
-        ? filterOutCurrentEpisode(nextEpisodes)
-        : nextEpisodes,
+      nextEpisodes,
     };
   }
 
-  async getTopics(prisonId) {
-    return this.#cmsApi.get(new TopicsQuery(prisonId));
+  async getTopics(establishmentName) {
+    return this.#cmsApi.get(new TopicsQuery(establishmentName));
   }
 
-  async getHomepage(prisonId) {
-    const homepages = await this.#cmsApi.get(new HomepageQuery(prisonId));
+  async getHomepage(establishmentName) {
+    const homepages = await this.#cmsApi.get(
+      new HomepageQuery(establishmentName),
+    );
     return homepages[0];
   }
+
+  async getNextEpisode(establishmentName, seriesId, seriesSortValue) {
+    const nextEpisodes = await this.#cmsApi.get(
+      new NextEpisodeQuery(
+        establishmentName,
+        seriesId,
+        seriesSortValue || undefined,
```

Comment:
Passed in `seriesSortValue` or undefined as the below filter in the query did not accept a `null` value when a next episode is not passed. 

`.addFilter('series_sort_value', seriesSortValue, '>')`